### ISSUE TITLE
Allow users to choose between paginated timelines and infinite scrolling

### ISSUE THREAD

**Issue by imacdonald** [pixelfed/pixelfed#325]
Regardless of browser (mostly on laptop but also tablet) when approaching page bottom, with continuous scrolling, the positioning jumps about 5 posts. You then have to scroll back up the page to see the missing posts.

A paginated timeline would be nice as an option.

**Comment by veb** [comment]
wouldn't it perhaps be better to make the continuous scrolling more efficient than to add two different options of scrolling through the timeline @imacdonald

**Comment by dansup** [comment]
@imacdonald Paginated timelines are already supported, they are for users that do not have javascript enabled. I could make it an option to disable in the settings if you have javascript enabled!

**Comment by imacdonald** [comment]
Excellent!
Presumably, logged-in users would need to have js enabled for tagging and description editing etc. An option in settings would be cool.

**Comment by trwnh** [comment]
A toggle has been added recently.
